skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -229,34 +240,59 @@ The following updates are applicable for TeamsOnly customers using Microsoft Tea
 - CountryAbbreviation
 - UsageLocation
 
-*Attributes renamed/replaced:*
-- ObjectId renamed to Identity
+*Attributes renamed*
+
 - FirstName renamed to GivenName
 - DirSyncEnabled renamed to UserDirSyncEnabled
 - MCOValidationErrors renamed to UserValidationErrors
-- OnPremSIPEnabled renamed to OnPremIsSipEnabled
 - Enabled renamed to IsSipEnabled
-- OnPremOptionFlags renamed to OnPremOptionFlag
-- OptionFlags renamed to OptionFlag
-- ProxyAddresses renamed to ProxyAddress
-- ShadowProxyAddresses renamed to ShadowProxyAddress
 - TeamsBranchSurvivabilityPolicy renamed to TeamsSurvivableBranchAppliancePolicy
 
-*New User Attributes*
+*Attributes that have changed in meaning/format*
 
-FeatureTypes – Array of unique strings specifying what features are enabled for a user (plan not displayed).
+OnPremLineURI: This attribute is used to refer to both:
+1. Previous LineURI set via OnPrem AD.
+2. Previous Direct Routing numbers assigned to users via Set-CsUser.
 
-*Deprecated input parameters*
+In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via OnPrem AD. Direct Routing numbers can be distinguished from Calling Plan users by looking at the FeatureTypes attribute.
```

Comment:
nit: did you mean 'for' calling plan users?

Reply:
No I meant we can distinguish between Direct Routing numbers and Calling Plan using FeatureTypes